lib/api_account.go

```diff
@@ -4,23 +4,21 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/gorilla/mux"
+
 	"boscoin.io/sebak/lib/block"
 	"boscoin.io/sebak/lib/common"
 	"boscoin.io/sebak/lib/error"
 	"boscoin.io/sebak/lib/observer"
-	"boscoin.io/sebak/lib/storage"
-	"github.com/gorilla/mux"
 )
 
 const GetAccountHandlerPattern = "/account/{address}"
 
-func GetAccountHandler(storage *sebakstorage.LevelDBBackend) http.HandlerFunc {
-
-	return func(w http.ResponseWriter, r *http.Request) {
-
+func (api NetworkHandlerAPI) GetAccountHandler(w http.ResponseWriter, r *http.Request) {
+	{
```

Comment:
Why the extra `{` ?

Reply:
@Geod24 https://github.com/bosnet/sebak/pull/279#discussion_r212880191

Reply:
https://github.com/bosnet/sebak/pull/279/files#r213527182